homeassistant/components/light/insteon_plm.py

```diff
@@ -0,0 +1,120 @@
+"""
+Support for INSTEON lights via PowerLinc Modem.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/insteon_plm/
+"""
+import logging
+import asyncio
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+from homeassistant.loader import get_component
+
+DEPENDENCIES = ['insteon_plm']
+
+MAX_BRIGHTNESS = 255
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
+    """Set up the INSTEON PLM device class for the hass platform."""
+    _LOGGER.info('Provisioning Insteon PLM Lights')
+
+    plm = hass.data['insteon_plm']
+
+    def async_plm_light_callback(device):
```

Comment:
Can you annotate this with `@callback` (from `homeassistant.core`). It improves readability.